proto/redpanda/api/dataplane/v1/user.proto

```diff
@@ -132,7 +132,7 @@ message DeleteUserResponse {}
 service UserService {
   option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_tag) = {
     name: "Redpanda Users"
-    description: "Manage Redpanda users. To manage access, see the Data Plane [ACL endpoints](https://docs.redpanda.com/api/cloud-dataplane-api/#get-/v1/acls)."
```

Comment:
Not sure if this is still correct? We also have the Security (roles) endpoints and I am not sure if we should be providing different guidance or pointing users to other docs. For example here are some of the authZ related docs we have in Cloud: https://docs.redpanda.com/redpanda-cloud/security/authorization/